sdk/communication/Azure.Communication.CallingServer/tests/CallConnection/CallConnectionLiveTests.cs

```diff
@@ -401,38 +351,53 @@ public async Task RunCreateAddCreateAudioRoutingRemoveHangupScenarioTests()
                 Assert.NotNull(addParticipantResult);
                 await WaitForOperationCompletion().ConfigureAwait(false);
 
+                // Add Participant To Default Audio Group
+                var addParticipantToDefaultAudioGroupResult = await AddParticipantToDefaultAudioGroupOperation(callConnection, userId).ConfigureAwait(false);
+                Assert.NotNull(addParticipantToDefaultAudioGroupResult);
+                await WaitForOperationCompletion().ConfigureAwait(false);
```

Comment:
let us not add this to this test. this test should only handle the renaming of AudioRoutingGroup to AudioGroup for Create.Get.Update,Remove - and i do not see Update being tested here, can you pl add that